docs/loader.md

```diff
@@ -187,12 +187,12 @@ Setup Instructions:
 1. Sign up for a Serverless account [here](https://app.serverless.com/)
 2. Install Serverless framework via command line `npm install -g serverless` to allow our Go code to interact with the Serverless CLI framework e.g. `sls deploy`
 3. Follow their [setup guide](https://www.serverless.com/framework/docs/getting-started) to link the respective cloud providers
+4. Install AWS CLI and Docker CLI (ensure that root permission is not required for `aws` and `docker` commands)
```

Comment:
Does it need to be done manually for each loader? If so, please make a script for automatic installation.